src/iotjs_binding.cpp

```diff
@@ -139,142 +115,126 @@ JObject::~JObject() {
 }
 
 
-static JRawValueType jnull_val = JVal::Null();
-static JObject jnull(&jnull_val);
-JObject& JObject::Null() {
-  return jnull;
+JObject JObject::Null() {
+  return JObject(JVal::Null(), false);
```

Comment:
This creates new a null JObject over and over again. Previous implementation used a singleton pattern. It seems a reasonable approach for null. (Even though the implementation of the pattern might be improved / made nicer.) What is the reason behind changing it?